pkg/logging/log.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+)
+
+func Info(output string, args ...interface{}) {
+	color.Blue("Info:")
```

Comment:
In the CI output, all the "Info:" show up on their own line. Is there a way to combine them on the same line as the rest of the info? The output looks awkward currently with no details on the same line as "Info:"

Reply:
agree, should be good now